### PR TITLE
Create sandboxes

### PR DESCRIPTION
**Pull request by thabarbados**
### Fix or Enhancement?
Add examples for codesandbox

- [x] All tests passed

### Environment
- OS: BigSur MacOs
- NPM Version: 6.14.15

### CODE REVIEW

File: sandboxes/bubble/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import Vue from 'vue'
+import App from '../src/App.vue'
```
Comment (dangreen) [inline sandboxes/bubble/index.js:2]:
```suggestion
import App from './src/App.vue'
```

Maybe there should be that path?

**Review by dangreen — CHANGES_REQUESTED** [review]

File: sandboxes/bubble/index.js (hunk 1 above)
Comment (thabarbados) [reply sandboxes/bubble/index.js]: sure. fix

File: sandboxes/bar/index.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import Vue from 'vue'
+import App from './src/App.vue'
+
+new Vue({
+  render: h => h(App)
+}).$mount('#app')
```
Comment (ilajosmanov) [inline sandboxes/bar/index.js:6]: I guess you can create a fabric for that

File: sandboxes/bar/src/components/barChart.vue [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      ],
+      datasets: [
+        {
+          label: 'Data One',
+          backgroundColor: '#f87979',
+          data: [40, 20, 12, 39, 10, 40, 39, 80, 40, 20, 12, 11]
+        }
+      ]
+    },
+    options: {
```
Comment (ilajosmanov) [inline sandboxes/bar/src/components/barChart.vue:36]: I know there are just examples, but I think some options/configs for chart we can freeze and put to computed

Comment (thabarbados) [reply sandboxes/bar/src/components/barChart.vue:36]: Evan [said](https://twitter.com/phiterf/status/1408039716524539910?s=12) it was overhead. I agree with him.

**Review by dangreen — APPROVED** [review]

**Pull request merged**